app/models/user.rb

```diff
@@ -59,4 +60,10 @@ def after_confirmation
       self.invitation = nil
     end
   end
+
+  def set_defaults
+    if self.permissions.empty?
+      self.permissions << VIEW_PERMISSION << ANNOTATE_PERMISSION
+    end
+  end
```

Comment:
Give this a try; I think it's the most modern method: https://stackoverflow.com/a/43484863/241735

Reply:
Ooooh, sweet! Will use that.